Feature: Update checker (#591)
paperless-ngx/paperless-ngx: docs/configuration.rst

        PAPERLESS_OCR_LANGUAGE=tur

    Defaults to none, which does not install any additional languages.

PAPERLESS_ENABLE_UPDATE_CHECK=<bool>

>     Enabling this option enables the check for available updates, which works by

Reviewer: ```suggestion
    Enable (or disable) the automatic check for available updates. This works by
```

Reviewer: To keep consistent with the other bools
